### DISCUSSION TITLE
Bun v1.1.40

### DISCUSSION THREAD

**Discussion by Jarred-Sumner** [oven-sh/bun#15835] [Announcements]
To install Bun v1.1.40
```bash
curl -fsSL https://bun.sh/install | bash
# or you can use npm
# npm install -g bun
```

Windows:
```bash
powershell -c "irm bun.sh/install.ps1|iex"
```

To upgrade to Bun v1.1.40:
```bash
bun upgrade
```

### Thanks to 4 contributors!

- [@190n](https://github.com/190n)
- [@dylan-conway](https://github.com/dylan-conway)
- [@jarred-sumner](https://github.com/jarred-sumner)
- [@riskymh](https://github.com/riskymh)

<hr /><em>This discussion was created from the release <a href='https://github.com/oven-sh/bun/releases/tag/bun-v1.1.40'>Bun v1.1.40</a>.</em>

**Comment by tenpaMk2** [comment]
Thank you developers❗️
My VSCode can stop break points now❗️
